fiftyone/utils/cvat.py

```diff
@@ -2523,6 +2524,11 @@ class CVATBackendConfig(foua.AnnotationBackendConfig):
         occluded_attr (None): an optional attribute name containing existing
             occluded values and/or in which to store downloaded occluded values
             for all objects in the annotation run
+        git_repo (None): the url of the git repository to link with the
```

Comment:
Let's use `git_repository` here. In general I'd like to avoid using abbreviations unless the parameter name would be realllly long. `attr` is a bit of a grandfathered in exception.